<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RadioPI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="SenderList">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Einen Sender aus der Liste w채hlen</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="__label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Sender w채hlen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="volPerc">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Lautst채rke: 0%</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="volume">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Lautst채rke anpassen</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSender">
    <property name="title">
     <string>Sender</string>
    </property>
    <addaction name="actionListe_bearbeiten"/>
   </widget>
   <addaction name="menuSender"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="toolTip">
    <string/>
   </property>
  </widget>
  <action name="actionVerbinden">
   <property name="text">
    <string>Verbinden</string>
   </property>
  </action>
  <action name="actionTrennen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Trennen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionListe_bearbeiten">
   <property name="text">
    <string>Liste bearbeiten</string>
   </property>
  </action>
  <action name="actionListe_exportieren">
   <property name="text">
    <string>Liste exportieren</string>
   </property>
  </action>
  <action name="actionListe_importieren">
   <property name="text">
    <string>Liste importieren</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
